app/database/admin/data_operator/index.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {MM_TABLES} from '@constants/database';
+import {
+    BatchOperations,
+    DBInstance,
```

Comment:
(Not important) - Just wanting to stick to a convetion; either `DBSomething` or `DatabaseSomething` rather than keep swapping between the two.